### DISCUSSION TITLE
Documentation website ideas / layout / content

### DISCUSSION THREAD

**Discussion by maartentamboer** [maartentamboer/X-Touch-Mini-FS2020#48] [General]
Let's discuss documentation ideas and content over here since the original issue was about LVARs (https://github.com/maartentamboer/X-Touch-Mini-FS2020/issues/39)
The website can be found over here: https://dev-tty.nl/X-Touch-Mini-FS2020/

**Comment by SQLSammy** [comment]
Just a 1st idea

Content 

1. Installation and configuration

2. general syntax elements
- variables
- multi actions, like 1 press, set more than 1 value
- conditions
- push types
- event types
- print
- data. methods and functions

3. general snippets by function  (f.e. set xtouch light)
4. general snippets with aircraft independent events/variables (f.e. COM, NAV, ADF, ....)
5. aircraft specific snippets

6. MobiFlight interaction
- installing mobiflight module tipps
- events.txt modifications

7. Additional ressources
- How to find events MSFS SDK documentation
- How to find events Mobiflight
- How to find events aircraft folder/code

**Reply by maartentamboer** [reply]
First parts of the documentation are there 😄 https://dev-tty.nl/X-Touch-Mini-FS2020/
I will continue tomorrow to hopefully at least finish the documentation of all the advanced features.

**Comment by rostykk** [comment]
0. Graphic layout for provided configs

**Reply by maartentamboer** [reply]
![image](https://user-images.githubusercontent.com/1395437/113412376-58eb1900-93b8-11eb-8881-aeb0c1866782.png)
Something like this? This was automatically generated by plantuml
